tests/test_server.py

```diff
@@ -44,6 +47,16 @@ def test_xeventlet(self):
         self.assertTrue(resp.ok)
         self.assertEqual(resp.content, b'<b>Hello xeventlet</b>!')
 
+    def test_simpl_server(self):
+        argv = ['server', '--quiet-server', '--port', str(get_free_port())]
+        proc = multiprocessing.Process(
+            target=simpl_cli.main, kwargs={'argv': argv})
+        proc.start()
+        time.sleep(.5)
```

Comment:
Is 0.5 seconds a guess on how long it will take the process to start? Maybe combining https://github.com/samstav/simpl/blob/simpl-server/tests/test_server.py#L112 with a timeout would be a more reliable approach? The test should not fail if the process takes longer than 0.5s and it should not wait idly for 0.5 seconds if the process is up in 10ms.